Sources/kitchen.js

```diff
@@ -195,7 +197,17 @@ function openTemplateFromXMLString(xmlString, presentationType) {
     presenterForType(presentationType).call(this, doc);
 }
 
+function reloadTab(atIndex) {
+    var tabItemToReload = tabItemDictionary[atIndex];
+    var feature = currentTab.parentNode.getFeature("MenuBarDocument");
+    if (feature) {
+        var docToReload = feature.getDocument(tabItemToReload);
```

Comment:
Just realized.
This just reloads same static xml that has been presented already.
I need to pass new xml to replace I guess.